superset/connectors/sqla/views.py

```diff
@@ -108,6 +115,23 @@ class TableColumnInlineView(CompactCRUDMixin, SupersetModelView):  # noqa
         'database_expression': _('Database Expression'),
         'type': _('Type'),
     }
+    validators_columns = {
+        'python_date_format': [
+            Regexp(
+                re.compile(
+                    r"""
+                    ^(
+                        epoch_s|epoch_ms|
+                        (?P<date>%Y(-%m(-%d)?)?)
+                        ([\sT](?P<time>%H(:%M(:%S(\.%f)?)?)?))?
```

Comment:
i trust that this is right, but maybe it's worth adding a comment explaining the regex?

Reply:
Done.